var/spack/repos/builtin/packages/bacio/package.py

```diff
@@ -37,6 +40,11 @@ class Bacio(CMakePackage):
 
```

Comment:
We need to declare a `shared` variant here. We can put the version restriction here as well. Something to the effect of `variant("shared", default=False, description="Build shared library", when="@2.6.0:")`. Spack is smart enough that down below when it runs `define_from_variant`, it will skip it if the condition doesn't match what's declared in the variant (i.e., it'll automatically skip setting `BUILD_SHARED_LIBS` if the version is older than 2.6.0).

Reply:
OK, take a look...

Reply:
I don't see any changes

Reply:
OK, my push didn't work. Now I've got it, try again...